File: ddd-application/src/main/java/victor/training/ddd/agile/repo/SprintRepo.java
package victor.training.ddd.agile.repo;

import victor.training.ddd.agile.entity.Sprint;
import victor.training.ddd.agile.common.CustomJpaRepository;

import java.util.List;

public interface SprintRepo extends CustomJpaRepository<Sprint, Long> {
    List<Sprint> findByProductId(Long id);
}

File: ddd-application/src/main/java/victor/training/ddd/agile/repo/SprintRepo.kt
package victor.training.ddd.agile.repo

import victor.training.ddd.agile.common.CustomJpaRepository
import victor.training.ddd.agile.entity.Sprint

interface SprintRepo : CustomJpaRepository<Sprint, Long> {
    fun findByProductId(productId: Long):List<Sprint>
}